packages/core/helper-plugin/lib/src/old/components/OverlayBlocker/index.js

```diff
@@ -0,0 +1,142 @@
+/*
+ *
+ * OverlayBlocker
+ *
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import cn from 'classnames';
+import Container from './Container';
+import Overlay from './Overlay';
+
+const DELAY = 1000;
+
+class OverlayBlocker extends React.Component {
```

Comment:
This component should not exist anymore.

Reply:
Sorry that was me, bad conflict fix